inc/thorvg.h

```diff
@@ -1177,6 +1201,49 @@ class TVG_EXPORT Picture final : public Paint
      */
     Result load(uint32_t* data, uint32_t w, uint32_t h, bool copy) noexcept;
 
+    /**
+     * @brief Sets the triangle mesh. Any existing mesh will be replaced by the new one.
+     * 
+     * If a mesh is provided, the transform property of the Picture will apply to the triangle mesh, and the
+     * image data will be used as the texture.
+     * 
+     * @param[in] triangles An array of Polygon objects (triangles) that make up the mesh. Must be at least 1.
+     * @param[in] count The number of Polygon objects (triangles) provided
+     * 
+     * @retval Result::Success When succeed.
+     * @retval Result::NonSupport If there was an error
+     * 
+     * @note The Polygons are copied internally, so modifying them after calling Mesh::mesh has no affect.
+     */
+    Result mesh(const Polygon* triangles, const uint32_t count) noexcept;
+
+    /**
+     * @brief Adds additional triangles to the mesh. It is safe to call this even if there is no mesh yet.
+     * 
+     * If a mesh is provided, the transform property of the Picture will apply to the triangle mesh, and the
+     * image data will be used as the texture.
+     * 
+     * @param[in] triangles An array of Polygon objects (triangles) that make up the mesh. Must be at least 1.
+     * @param[in] count The number of Polygon objects (triangles) provided
+     * 
+     * @retval Result::Success When succeed.
+     * @retval Result::NonSupport If there was an error
```

Comment:
Looking at the code, it seems that `Result::NonSupport` does not returned.
Could you please add an explanation for returning `Result::InsufficientCondition`?

Reply:
Thank you for picking this up! I will change this to `Result::InvalidArguments` and add an explanation.

Reply:
Done